content/docs/2.13/operate/events.md

```diff
@@ -82,3 +82,8 @@ Here is an overview of the supported destinations:
     http:
       uri: http://foo.bar  #An http endpoint that can receive cloudevent
 ```
+### Supported Event List
+| Event                                 |  Description                                                                                                                 | 
+| ------------------------------------- |  --------------------------------------------------------------------------------------------------------------------------- | 
+| `ScaledObjectReady`                   | On the first time a ScaledObject is ready, or if the previous ready condition status of the object was `Unknown` or `False` |  
+| `ScaledObjectCheckFailed`             | If the check validation for a ScaledObject fails |      
```

Comment:
Are these really the types? Because these are not per the spec defined on https://github.com/kedacore/keda/issues/3522